app/Services/MediaWikiHostResolver.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Services;
```

Comment:
I'm unclear about the introduction of this `Service` namespace and location, this could be confused with the Laravel concept of Service Providers https://laravel.com/docs/10.x/providers

I think `App\Helper` would be more appropriate https://github.com/wbstack/api/tree/main/app/Helper